src/main/java/PaidByUser.java

```diff
@@ -0,0 +1,19 @@
+package src.main.java;
+
+public class PaidByUser extends User {
+    private String paymentMethod;
```

Comment:
consider using `enum` for payment method.

Reply:
Thanks for the idea. I have committed this improvement of using an enum to define payment method instead of a string.